doc/ale-cairo.txt

```diff
@@ -13,3 +26,5 @@ g:ale_cairo_starknet_executable               *g:ale_cairo_starknet_executable*
   Overrides the starknet-compile binary after installing the cairo-language.
 
   For more information read 'https://starknet.io/docs/quickstart.html'
+
+===============================================================================
```

Comment:
Check other files to see if this should be there or not. Follow the most common example. I can't remember what is best.

Reply:
From what I can tell in `/doc/` all files end with a separator so i just left it as is.